metaflow/util.py

```diff
@@ -50,6 +50,19 @@ def unquote_bytes(x):
         return unquote(to_unicode(x))
 
 
+if sys.version_info.major >= 3 and sys.version_info.minor >= 7:
```

Comment:
Where is `namedtuple_with_defaults` used in the PR?

Reply:
L15 of s3.py.